CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.4.15
+### Fixes
+- Fix pagination issues for target groups, users, and IP blocks
```

Comment:
```suggestion
- Increase max result limit of data sources for target groups(200) and ip blocks(1000). Workaround for pagination issues.
- Add email filter for user data source. Fixes pagination issues for users.
```

Reply:
@cristiGuranIonos done, thanks for the suggestion!